protocol/v2/ssv/runner/runner_state.go

```diff
@@ -18,25 +17,24 @@ type State struct {
 	PostConsensusContainer *ssv.PartialSigContainer
 	RunningInstance        *instance.Instance
 	DecidedValue           []byte //spectypes.Encoder
-	// CurrentDuty is the duty the node pulled locally from the beacon node, might be different from decided duty
+	// StartingDuty is the duty the node pulled locally from the beacon node, might be different
+	// from the actual duty operators have decided upon.
 	StartingDuty spectypes.Duty `json:"StartingDuty,omitempty"`
```

Comment:
> might be different from decided duty

Is the only reason it can be different - because other operators might have gotten different data from their beacon nodes (and hence decided value for operator cluster is different), or there is something else ?